src/USER-QUIP/pair_quip.cpp

```diff
@@ -202,6 +213,8 @@ void PairQUIP::compute(int eflag, int vflag)
 void PairQUIP::settings(int narg, char **arg)
 {
   if (narg != 0) error->all(FLERR,"Illegal pair_style command");
+  if (strncmp(force->pair->style,"hybrid",6) == 0)
+    error->all(FLERR,"Pair style quip is not compatible with hybrid styles");
```

Comment:
Is there a specific reason for this? I've been using quip potentials as part of a `hybrid/overlay` pair style without any apparent problems.